addons/mail/static/src/core/common/message_model.js

```diff
@@ -533,7 +537,7 @@ export class Message extends Record {
         attachments = [],
         { mentionedChannels = [], mentionedPartners = [], mentionedRoles = [] } = {}
     ) {
-        if (convertBrToLineBreak(this.body) === body && attachments.length === 0) {
+        if (isHtmlTextuallyEqual(body, this.body) && attachments.length === 0) {
```

Comment:
Why do we even need to compare them this way?
If the body is not the same HTML-wise, we probably need to send the new one.